packages/core/src/tooltip/useTooltip.ts

```diff
@@ -108,8 +108,9 @@ export function useTooltip(props?: UseTooltipProps) {
     ref: arrowRef,
     context,
     style: {
-      left: arrowX ? `${arrowX}px` : "",
-      top: arrowY ? `${arrowY}px` : "",
+      position: strategy,
+      left: arrowX ?? 0,
```

Comment:
This was needed to fix the positioning of Tooltips within the Portal. If not the arrow and body would be weird on the first render